intro-to-R-tidyverse/04c-intro_to_tidyverse_exercise-part-2.Rmd

```diff
@@ -151,7 +151,7 @@ the plot completely to something else!
 # Ideas to get your juices flowing: 
 # - Add a jitter layer over the boxplot
 # - Play with the color of the points!
-# - Use ggthemes to play with the aesthetics
+# - Use ggplot2::theme to play with the aesthetics
```

Comment:
This should be updated to match the change made in the solved version of the notebook [here](https://github.com/AlexsLemonade/exercise-notebook-answers/blob/1c4447bd60f03d0c7c316b86aef9d2da1c7ecea4/intro-to-r-tidyverse/04c-intro_to_tidyverse_exercise-part-2_solved.Rmd#L190) (the change referenced [in this comment](https://github.com/AlexsLemonade/exercise-notebook-answers/pull/23#discussion_r430439177)).